File: app/src/main/java/org/opencv/samples/tutorial2/Mainactivity.java
package org.opencv.samples.tutorial2;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.support.annotation.Nullable;
import android.util.Log;

public class Mainactivity extends Activity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        startActivityForResult(new Intent(this,DetectActitvity.class),100);
    }

    @Override
    protected void onDestroy() {
        super.onDestroy();
        System.exit(0);
    }

    @Override
    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
        super.onActivityResult(requestCode, resultCode, data);
        Log.e("onActivityResult ", "" + resultCode);
        Log.e("onActivityResult ", "" + requestCode);
        if(resultCode == -2)
            finish();
        if(requestCode == 100 && resultCode == RESULT_OK){
            Handler h = new Handler();
            h.postDelayed(new Runnable() {
                @Override
                public void run() {
                    startActivityForResult(new Intent(Mainactivity.this,DetectActitvity.class),100);
                }
            },1000);

        }
    }
}

File: app/src/main/java/org/opencv/samples/tutorial2/MainActivity.kt
package org.opencv.samples.tutorial2

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import com.ut.sdk2.R
import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.activity_main.view.*

class MainActivity :Activity() {
    var checked = R.id.internal
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        b640.setOnClickListener {
            var intent = Intent(this,DetectActitvity::class.java)
            intent.putExtra("px",640)
            intent.putExtra("store",checked)
            startActivity(intent)
        }
        b720.setOnClickListener {
            var intent = Intent(this,DetectActitvity::class.java)
            intent.putExtra("px",720)
            intent.putExtra("store",checked)
            startActivity(intent)
        }
        back.setOnClickListener {
            finish()
        }
        radio_g.setOnCheckedChangeListener { group, checkedId ->
            checked = checkedId
        }
    }

}